doc/content/examples/example9.md

```diff
@@ -0,0 +1,89 @@
+
+!listing examples/ex09/Structure_Response.i
+
+!media media/examples/ex09_Structure.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Structure
+       caption=Acceleration response spectra of the fixed base structure subjected to the Ormsby Wavelet.
+
+### Site response analysis
+
+The input file for simulating the individual soil response is presented below. The uniform soil domain has a shear wave velocity of $165$ $m/s$. Although the soil is uniform throughout, the `UniformLayerAuxKernel` is used to specify the top surface of the soil. This is necessary when using the `ComputeIsotropicElasticityTensorSoil` material. Alternatively, if the `ComputeIsotropicElasticityTensor` is used for a uniform soil, the `UniformLayerAuxKernel` is not necessary. In addition, periodic boundary conditions are applied to the two opposite faces of the soil domain which are perpendicular to the Z axis to ensure that they move in unison. This achieved using the `Periodic` option in `BCs`. The Ormsby Wavelet is applied at the bottom of the soil block and the response is recorded at the top. The 1% damped acceleration response spectrum at the top of the soil is presented below. The periods at which the response spectrum peaks are the fundamental periods of the soil domain. The periods inferred from this figure are very similar to those predicted by the theoretical solution $4H/((2n-1)~Vs)$, where $H$ is the depth of soil and $n$ is the index of the period of interest.
+
+!listing examples/ex09/Soil_Response.i
+
+!media media/examples/ex09_Soil.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Soil
+       caption=Acceleration response spectra of the soil subjected to the Ormsby Wavelet.
+
+### Soil Structure Interaction analysis
+
+The input file for simulating the SSI response is presented below. The amplification ratio of the structural response spectra at 10m height from the free-field soil is computed. Free-field soil is the soil region located far away from the structure, which in this case is one of the top end most nodes of the domain. [fig:SSI_Amp] presents the amplification ratio as a function of the period. This amplification ratio shows a good agreement with that using LS-Dyna computed by [!citet](BolisettiThesis) as presented in [fig:SSI_Amp_Valid].
```

Comment:
I would argue that this is not a good agreement. 

I would actually suggest removing the comparison with the SDEE paper. The comparison shows that SASSI and LS-DYNA, which are completely different software have similar results, which are different from MASTODON. This is not good. Of course, the reason is that the model domain sizes are different, in addition to other differences.